### PR TITLE
Update customize-your-organization-theme.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed repeating words.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7424

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/customize-your-organization-theme.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ You can create up to four additional group themes.
 
 ## Logos: Specify your theme logos
 
-On the **Logos** page, you can you can add your logos, and specify the URL where users will navigate to, when they select the logo.
+On the **Logos** page, you can add your logos, and specify the URL where users will navigate to, when they select the logo.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/customize-your-organization-theme.md]:
```suggestion
On the **Logos** page, you can add your logos, and specify the URL where users will navigate to when they select the logo.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**